### PR TITLE
[DPE-2164] Add Spark SQL into Charmed Spark Rock image

### PR DESCRIPTION
**Pull request by theoctober19th**
This PR adds an entrypoint to `spark-sql` binary in Charmed Spark Rock image.

### CODE REVIEW

File: rockcraft.yaml [hunk 1]
```diff
@@ -147,7 +147,7 @@ parts:
       - python3-pip
     overlay-script: |
       mkdir -p $CRAFT_PART_INSTALL/opt/spark8t/python/dist
-      pip install --target=${CRAFT_PART_INSTALL}/opt/spark8t/python/dist  https://github.com/canonical/spark-k8s-toolkit-py/releases/download/v0.0.3/spark8t-0.0.3-py3-none-any.whl
+      pip install --target=${CRAFT_PART_INSTALL}/opt/spark8t/python/dist  git+https://github.com/theoctober19th/spark-k8s-toolkit-py.git@spark-sql
```
Comment (theoctober19th) [inline rockcraft.yaml]: TODO: This is to be updated once we have a new release of `spark8t` library.

**Review by welpaolo — DISMISSED** [review]
LGTM.

**Review by deusebio — DISMISSED** [review]
LGTM!!! Great job!

File: rockcraft.yaml (hunk 1 above)
Comment (theoctober19th) [reply rockcraft.yaml]: Updated the URL to point to spark8t `v0.0.5`.

**Review by welpaolo — APPROVED** [review]

**Review by deusebio — APPROVED** [review]

**Pull request merged**